pipeline.jsonc

```diff
@@ -30,7 +30,8 @@
         },
         "configuration":
         [
-            "170=v1.7.0"
+            "170=v1.7.0",
+            "209=v2.0.9"
```

Comment:
What prevents the protocol features multiversion test (`nodeos_multiple_version_protocol_feature_mv_test`) from testing against this new version (`v2.0.9`)?

Reply:
it should work. however is this something we want? @brianjohnson5972 ?

Reply:
I think it should work.  Turn it on and then we will investigate if it fails.

Reply:
so after looking deeper into it..
version 1.7.0 was last without preactivate protocol feature. as per the test it is supposed to go out of sync after activation. So this test supposed to fail with 2.0.9 and has no sense with it.
having that we don't have much 1.7.0 we may want to deactivate this test. Instead we will either maintain extra data multiversion test or create another to test our current master branch with latest release.
based on discussion with Santhosh and Brian

Reply:
> having that we don't have much 1.7.0 we may want to deactivate this test.

I don't think we want to delete it. I asked about this on the video call where @arhag presented on the multiversion test, and it sounded like we want to keep it for now because it can still catch breaking changes with the protocol features, but he can speak more on that if he wants to.

We have `v1.7.0` builds and binaries safely stored in the cloud and we can keep them more or less forever so there is no technical reason to push for deleting this test because it is old, or anything like that.

> create another to test our current master branch with latest release.

I thought that is what this pull request is for. I must not understand, can you expand on these two suggestions a bit?

Reply:
@dimas1185 answered my question via IM. There is a second definition in the protocol features test [here](https://github.com/EOSIO/eos/blob/74b0ac7b3f3885e82f86c8d0681b71033e21bbe4/tests/nodeos_multiple_version_protocol_feature_test.py#L87) and in the new test [here](https://github.com/EOSIO/eos/blob/74b0ac7b3f3885e82f86c8d0681b71033e21bbe4/tests/nodeos_extra_packed_data_test.py#L88) which defines which versions the tests are running against. I did poke around the code looking for this and skimmed the Python, but I had missed these definitions and believed one or both tests were behaving in an unexpected way, hence my reservations.